vaccine_feed_ingest/runners/ky/govstatus/normalize.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import logging
+import pathlib
+import sys
+
+from vaccine_feed_ingest_schema import schema  # noqa: E402
+
+logger = logging.getLogger(__name__)
+
+_source_name = "ky:govstatus"
```

Comment:
minor: Constants in python are normally defines in ALL_CAPS: https://www.python.org/dev/peps/pep-0008/#constants

Reply:
Done.